lib/model-builder.js

```diff
@@ -342,6 +342,21 @@ ModelBuilder.prototype.define = function defineClass(className, properties, sett
     });
   }
 
+  // updateOnly property is added to indicate that this property will appear in
+  // the model for update/updateorcreate operations but and not for create operation.
+  var forceId = ModelClass.definition.settings.forceId;
```

Comment:
@bajtos ModelClass.definition.settings.forceId and ModelClass.settings.forceId are exactly the same. Anyway changing to ModelClass.settings.forceId